### PR TITLE
Shrink docker image

### PR DESCRIPTION
**Pull request by joejulian**

### CODE REVIEW

**Comment by joejulian** [comment]
This is being tested by samsung-cnct/k2#482.

The test process is:
* commit & push k2-tools changes to my branch
* `docker build -t joejulian/k2-tools-test:$(git rev-parse HEAD) . && docker push joejulian/k2-tools-test:$(git rev-parse HEAD)`
* in k2, edit the Jenkinsfile and replace `quay.io/samsung_cnct/k2-tools:latest` with `docker.io/joejulian/k2-tools-test:` and the sha from the previous `git rev-parse HEAD`.
* add, commit, and push the change
* open a k2 PR (tagged "do not merge") with the edited Jenkinsfile.

File: Dockerfile [hunk 1]
```diff
@@ -36,20 +36,16 @@ ENV     GO15VENDOREXPERIMENT 1
...
-        # install all python packages
-RUN     pip install -r /requirements.txt
+# wget
+RUN     apk update && apk add ca-certificates wget &&  rm -rfv /var/cache/apk/*
 
 # Terraform
 
         #Installing Terraform binary
-RUN     curl -O https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+RUN     wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
```
Comment (coffeepac) [inline Dockerfile]: does this mean we no longer need to have curl installed?

Comment (joejulian) [inline Dockerfile]: No need and, indeed, it's not.

Comment (coffeepac) [inline Dockerfile]: HAH!  yes, I see its been removed now.

**Comment by coffeepac** [comment]
@joejulian for testing make sure you purge the kubernetes stored templates.  I haven't sorted that out what's wrong with that yet.

**Comment by joejulian** [comment]
I'm hoping that by using the sha instead of "latest" and changing the Jenkinsfile, it should invalidate the cache if I change that sha.

**Pull request merged**